pkg/autoscaler/autoscaler/util.go

```diff
@@ -344,3 +344,13 @@ func autoRulesToStrategyRule(component string, rules map[corev1.ResourceName]v1a
 	}
 	return result
 }
+
+func checkAutoscalingComponent(tas *v1alpha1.TidbClusterAutoScaler, component string) bool {
```

Comment:
How about putting this logic in line 99?

Reply:
line 99 in which file?

Reply:
in `pdplan_autoscaler.go`

Reply:
the function will be also used when creating autoscaling clusters. do you mean we put this function in that file?